cmd/root.go

```diff
@@ -115,6 +115,10 @@ func rootRunE(cmd *cobra.Command, args []string) error {
 		err = fmt.Errorf("files with violations: %d", violations)
 	}
 
+    if violations == 0 {
+        fmt.Println("No violations found. Stay woke \u270a")
+    }
```

Comment:
I'm surprised golangci didn't pick up the spaces here. Go code should be indented with tabs	(https://golang.org/doc/effective_go.html#formatting). Would you mind updating?